metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchQueryBuilder.java

```diff
@@ -120,7 +120,8 @@ private QueryBuilder buildInternalQuery(@Nullable QueryConfiguration customQuery
           .map(this::getStandardFields)
```

Comment:
Is this the only use of getStandardFields? I feel like the reduction should not rely on the various usages implementing the average. I'd maybe create a new function that is like getStandardFields(List<EntitySpec> specs) instead of a single spec. Unit test that function.